common/github.com/deepaksibm/vpc-go-sdk-beta/common/headers.go

```diff
@@ -0,0 +1,53 @@
+package common
+
+import (
+	"fmt"
+	"runtime"
+)
+
+const (
+	HEADER_NAME_USER_AGENT = "User-Agent"
+
+	SDK_NAME = "vpc-go-sdk"
```

Comment:
Is this right ?

Reply:
ignore changes in temporary sdk folders as these will be removed while raising upstream pr